README.rst

```diff
@@ -454,16 +454,184 @@ template ``context_processors`` in ``settings.py`` as shown below.
                 ],
                 'context_processors': [
                     # ... other context processors ...
-                    'openwisp_utils.admin_theme.context_processor.menu_items'    # <----- add this
+                    'openwisp_utils.admin_theme.context_processor.menu_groups'    # <----- add this
                 ],
             },
         },
     ]
 
-If you need to define custom menu items, see:
-`OPENWISP_ADMIN_MENU_ITEMS <#openwisp_admin_menu_items>`_.
+``register_menu_group``
+^^^^^^^^^^^^^^^^^^^^^^^^
 
-Users will only be able to see menu items for objects they have permission to either view or edit.
+It allows us to register different menu item or group at different position in the ``Main Navigation Menu``.
+
+**Syntax:**
+
+.. code-block:: python
+
+    register_menu_group(position, config)
+
++--------------------+-------------------------------------------------------------+
+| **Parameter**      | **Description**                                             |
++--------------------+-------------------------------------------------------------+
+| ``position``       | (``int``) Position of the group or item.                    |
++--------------------+-------------------------------------------------------------+
+| ``config``         | (``dict``) Configuration of goup or item.                   |
++--------------------+-------------------------------------------------------------+
+
+Code example:
+
+.. code-block:: python
+
+    from openwisp_utils.admin_theme.menu import register_menu_group
+
```

Comment:
You know what would be great, to import the `ugettext_lazy as _` function here and then use it to flag strings as translatable, that way I can avoid having to repeat to everyone to flag strings as translatable when they register menu items :joy: .

Reply:
Thank you for letting me know, please don't do that, because Gettext scans the code to look for strings that have to be translated, if we do it in that way gettext will not be able to add those strings to the .po file (a file which lists all the strings that are translatable and allows translators to translate them to a specific language, see the [django po files for reference](https://github.com/django/django/blob/main/django/contrib/admin/locale/hi/LC_MESSAGES/django.po), in the near future we'll have translations for openwisp too).